### PR TITLE
Better Messaging about Search-by-SMS

### PR DESCRIPTION
**Pull request by DanielCoulbourne**
Can someone sanity check this for me? Just getting my feet wet.

### CODE REVIEW

**Review by gcavanunez — APPROVED** [review]

File: app/Events/PhoneNumberQueried.php [hunk 1]
```diff
@@ -17,12 +17,18 @@ public static function webhook(Request $request, SmsCommand $command): string
     {
         // TODO: Account for numbers with spaces :/
```
Comment (hans-lts) [inline app/Events/PhoneNumberQueried.php:18]: the `phone_number` helper handles this

**Review by hans-lts — APPROVED** [review]
Looks good

File: app/Events/PhoneNumberQueried.php (hunk 1 above)
Comment (DanielCoulbourne) [reply app/Events/PhoneNumberQueried.php:18]: @hans-lts can you make a code suggestion? I just moved existing code.

File: app/Events/CheckedInViaSms.php
Comment (jdiddydave) [inline app/Events/CheckedInViaSms.php:1]: I think it makes sense to use quotes around "SEARCH" like we do around "CANCEL"

File: app/Http/Controllers/Webhooks/TwilioWebhookController.php
Comment (jdiddydave) [inline app/Http/Controllers/Webhooks/TwilioWebhookController.php:1]: I think it makes sense to use quotes around "SEARCH" like we do around "UPDATE"

Maybe:
'To send updates on DisasterCheckin site to anyone who knows your number, start your msg with "UPDATE" (UPDATE I am OK); use "SEARCH" to find others (SEARCH 8285550000)',

File: app/Events/PhoneNumberQueried.php (hunk 1 above)
Comment (hans-lts) [reply app/Events/PhoneNumberQueried.php:18]: 😬 I may have been a little too hasty with that comment... I see now the phone number wouldn't even make it to the helper function due to `explode`.

My tests for the number with spaces fail and I'm a little too tired to get it handled right now. I will re-approach in the morning.

**Pull request merged**